src/Homie/Config.cpp

```diff
@@ -49,6 +47,17 @@ bool Config::load() {
   JsonObject& parsedJson = jsonBuffer.parseObject(buf);
```

Comment:
`jsonBuffer` is a local variable, so  you create a reference to something internal to this local variable.
The local variable gets destroyed when leaving `Config::load()`, this (hopefully ;-) frees the buffer of `parsedJson`.